src/Commands/Commands/Components/Command.cs

```diff
@@ -314,18 +311,9 @@ public string GetFullName(bool includeArguments)
     public string GetFullName()
         => GetFullName(true);
 
-    /// <inheritdoc />
-    public float GetScore()
-    {
-        var score = 1.0f;
-
-        foreach (var parameter in Parameters)
-            score += parameter.GetScore();
-
-        score += Attributes.FirstOrDefault<PriorityAttribute>()?.Priority ?? 0;
 
-        return score;
-    }
+    /// <inheritdoc />
+    public float GetScore() => 1 + Parameters.Sum(p => p.GetScore()) + Attributes.FirstOrDefault<PriorityAttribute>()?.Priority ?? 0;
```

Comment:
This will throw out of range when there are no parameters, so .DefaultIfEmpty must be used. This does not improve the old functionality. I also prefer newlining expressions format as shown in GetFullName()